site/snippets/react/useExportAccount.tsx

```diff
@@ -0,0 +1,23 @@
+import { useExportAccount } from "@alchemy/aa-alchemy/react";
+
+export function ComponentWithExportAccount() {
+  // Assumes the app has context of a signer with an authenticated user
+  const { exportAccount, isExported, isExporting, ExportAccountComponent } =
+    useExportAccount({
+      iframeContainerId: "turnkey-export-wallet-container-id",
+      iframeElementId: "turnkey-export-wallet-element-id",
```

Comment:
yea would be great to make this whole thing optional so that a user can just use it as `useExportAccount()`